src/Microsoft.AspNet.FileProviders/Implementation/Helper.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+
+namespace Microsoft.AspNet.FileProviders
+{
+    public class Helper
+    {
+        internal static bool CanBeServed(FileSystemInfo fileSystemInfo)
```

Comment:
`CanBeServed` seems orthogonal to the use case. Isn't this just saying whether the file is hidden or not?